src/components/Modals/Onboarding/OnboardingTransfer.vue

```diff
@@ -52,6 +52,10 @@
             >here</a
           >.
         </div>
+        <div class="onboarding-transferCreditDescription">
+          *Some transfer credits may incorrectly apply to requirements. We are working to fix these
```

Comment:
I'm not a designer, but I'm not a big fan of the * before the warning (and before the warning above). The text is already small and gray, so I already recognize it as "extra information". Having two `*`s is a little intrusive and uneven IMO.